recipes/einops/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "einops" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4fd64864fcb8159074da3213b9327c242536784416cbf423745ef8579850d30b"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

This can be `noarch: python` and the pip arguments are now included by default.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
  noarch: python
```